projects/core/src/lib/map/map-info/map-resolution-extent-overlay.component.ts

```diff
@@ -0,0 +1,61 @@
+import {ChangeDetectionStrategy, Component, Input} from '@angular/core';
+import {ProjectService} from '../../project/project.service';
+import {MapService} from '../map.service';
+import {BehaviorSubject, combineLatestWith, Observable} from 'rxjs';
+import {map} from 'rxjs/operators';
+import {SpatialReferenceService} from '../../spatial-references/spatial-reference.service';
+
+/**
+ * The `geoengine-map-resolution-extent-overlay` displays information about the resolution and extent of the visible map(s).
+ */
+@Component({
+    selector: 'geoengine-map-resolution-extent-overlay',
+    templateUrl: 'map-resolution-extent-overlay.component.html',
+    styleUrls: ['map-resolution-extent-overlay.component.scss'],
+    changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class MapResolutionExtentOverlayComponent {
+    @Input()
+    public bottom!: number;
+
+    highPrecision: BehaviorSubject<boolean> = new BehaviorSubject(false);
+    private static readonly lowNumFractions = 2;
+    private static readonly highNumFractions = 12;
+
+    constructor(
+        private mapService: MapService,
+        private spatialReferenceService: SpatialReferenceService,
+        private projectService: ProjectService,
+    ) {}
+
+    togglePrecision(): void {
```

Comment:
can you show a hand cursor when hovering the overlay so that users know something happens on click?

Reply:
https://github.com/geo-engine/geoengine-ui/pull/720/commits/fca7837d75a4889040b71ba97919799f3f9f2f77